### DISCUSSION TITLE
Multiple managment servers

### DISCUSSION THREAD

**Discussion by SviridoffA** [apache/cloudstack#9457] [Q&A] [answered]
I would be very grateful if someone could clarify the following question for me. I have a MySQL cluster with a virtual IP (pacemaker / corosync), and three management servers. I configured the first server via cloudstack-setup-databases then cloudstack-setup-management, according to the manual. Then I repeated the procedure on the another two management servers. When executing cloudstack-setup-databases, I specified the -i key and specified my IP on each server. After completing the configuration, everything seems to work, the changes are displayed on all management nodes. But if you go to Infrastracture - Management servers, only the last configured server is displayed. I would be very grateful if someone could suggest the correct option for configuring a scheme with several management servers.

**Comment by rajujith** [comment, marked as answer]
use the '--deploy-as' with the cloudstack-setup-databases only on the first management server.

**Comment by weizhouapache** [comment]
did you use "--deploy-as" in additional management servers ?

which cloudstack version do you use ?

**Comment by SviridoffA** [comment]
yes I use it... I was not paying attention, I shouldn't have used --deploy-as on additional servers. Thank you all for help!!!
